janusgraph-solr/pom.xml

```diff
@@ -56,19 +54,27 @@
             <groupId>org.apache.solr</groupId>
             <artifactId>solr-solrj</artifactId>
             <version>${solr.version}</version>
+            <exclusions>
+                <!-- Use jcl-over-slf4j from gremlin-core -->
+                <exclusion>
+                    <groupId>org.slf4j</groupId>
+                    <artifactId>jcl-over-slf4j</artifactId>
+                </exclusion>
+            </exclusions>
         </dependency>
         <dependency>
             <groupId>org.apache.solr</groupId>
```

Comment:
Yes, but this was only a half of the issues behind #310. 
If you align lucene versions correctly the error will be gone, but you still add a truckload of extra solr-core deps like servlet-api, jetty, restlet etc., which to me do not seem needed at all.